<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnifiedTaggingPanel</class>
 <widget class="QWidget" name="UnifiedTaggingPanel">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="contentsMargins">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="target_label">
     <property name="text">
      <string>Selected Target: None</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TagInputWidget" name="tag_input_widget"/>
   </item>
   <item>
    <widget class="BatchTaggingOptionsWidget" name="batch_options_widget"/>
   </item>
   <item>
    <widget class="QPushButton" name="save_tags_button">
     <property name="text">
      <string>Save Tags</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagInputWidget</class>
   <extends>QWidget</extends>
   <header>widgets.tag_input_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatchTaggingOptionsWidget</class>
   <extends>QWidget</extends>
   <header>widgets.batch_tagging_options_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
